src/chapters/css/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: CSS
+==============
+
+We have built a website for you to test your CSS knowledge on. Check out the `repl.it <https://repl.it/@launchcode/CSSExercises/>`_!
+
+For the exercises, add the following style rules to ``style.css``:
+
+1. Change the background color to yellow.
+2. Change all paragraph text color to red.
+3. Change all ``h1`` to 36 px font size.
```

Comment:
How do you feel about introducing responsive text sizing as a subset of this?  I remember an exercise in LC101 where we changed the window size to see how the text responded using ``36 px`` vs. something like ``2.5vw``.

Reply:
We didn't cover this in the chapter.